demos/motmetrics4norfair/README.md

```diff
@@ -21,4 +21,8 @@ Given that the ground truth files for the testing set are not publicly available
 2. Display the motmetrics4norfair instructions: 
 ```bash
     python motmetrics4norfair.py --help 
-``` 
\ No newline at end of file
+``` 
+
+## Remarks
+
+Hyperparameters chosen on this demo are not optimal for every metric. If you want to decrease the ID switches (`IDs`), you may use a higher margin between `hit_inertia_min` and `hit_inertia_max`.
```

Comment:
"Hyperparameters where tuned for reaching a high MOTA on this dataset. They may not be ideal for more general use cases, use the default hyperparameters for those. ID switches suffer specially due to this optimization. If you want to improve ID switches use a higher margin between `hit_inertia_min` and `hit_inertia_max`, or just use the default hyperparameters"

Reply:
Ok

Reply:
Same typo here in "where tuned".